support-frontend/assets/pages/supporter-plus-landing/components/__snapshots__/paymentTsAndCs.test.tsx.snap

```diff
@@ -16,6 +16,8 @@ exports[`Payment Ts&Cs Snapshot comparison paymentTs&Cs render product NationalD
 
-exports[`Payment Ts&Cs Snapshot comparison paymentTs&Cs render product SupporterPlus for region Monthly correctly 1`] = `"If you pay at least £12 per month, you will receive the All-access digital benefits on a subscription basis. If you increase your payments per month, these additional amounts will be separate monthly voluntary financial contributions to the Guardian. The All-access digital subscription and any contributions will auto-renew each month. You will be charged the subscription and contribution amounts using your chosen payment method at each renewal unless you cancel. You can cancel your subscription or change your contributions at any time before your next renewal date. If you cancel within 14 days of taking out a All-access digital subscription, you’ll receive a full refund (including of any contributions) and your subscription and any contribution will stop immediately. Cancellation of your subscription (which will also cancel any contribution) or cancellation of your contribution made after 14 days will take effect at the end of your current monthly payment period. To cancel, go to Manage My Account or see our Terms.By proceeding, you are agreeing to our Terms and Conditions. To find out what personal data we collect and how we use it, please visit our Privacy Policy.All card payments are powered by Stripe. Read the Stripe Privacy Policy and Terms and conditions."`;
+exports[`Payment Ts&Cs Snapshot comparison paymentTs&Cs render product SupporterPlus for region Monthly correctly 1`] = `"If you pay at least £12 per month, you will receive the All-access digital benefits on a subscription basis. If you increase your payments per month, these additional amounts will be separate monthly voluntary financial contributions to the Guardian. The All-access digital subscription and any contributions will auto-renew each month. You will be charged the subscription and contribution amounts using your chosen payment method at each renewal unless you cancel. You can cancel your subscription or change your contributions at any time before your next renewal date. If you cancel within 14 days of taking out a All-access digital subscription, you’ll receive a full refund (including of any contributions) and your subscription and any contribution will stop immediately. Cancellation of your subscription (which will also cancel any contribution) or cancellation of your contribution made after 14 days will take effect at the end of your current monthly payment period. To cancel, go to Manage My Account or see our Terms. By proceeding, you are agreeing to our Terms and Conditions. To find out what personal data we collect and how we use it, please visit our Privacy Policy.All card payments are powered by Stripe. Read the Stripe Privacy Policy and Terms and conditions."`;
```

Comment:
`for region Monthly` doesn't sound quite right here :-) I don't think it's your PR which has added this, but the snapshot changing made me notice it. I think the interpolation in the test description must not be quite right.

Reply:
Its a missing space before `By proceeding`  in the snapshot that was updated for I think (have a look at the copy)

Reply:
Chatted IRL and we'll fix this in a separate PR (it creates too much noise here)